src/core/sys/dragonflybsd/sys/link_elf.d

```diff
@@ -75,7 +75,7 @@ struct dl_phdr_info
 
 private alias int function(dl_phdr_info*, size_t, void *) dl_iterate_phdr_cb;
 private alias int function(dl_phdr_info*, size_t, void *) @nogc dl_iterate_phdr_cb_ngc;
-extern int dl_iterate_phdr(dl_iterate_phdr_cb __callback, void*__data);
-extern int dl_iterate_phdr(dl_iterate_phdr_cb_ngc __callback, void*__data) @nogc;
-extern int _rtld_addr_phdr(const void*, dl_phdr_info*) @nogc;
-extern int _rtld_get_stack_prot() @nogc;
+int dl_iterate_phdr(dl_iterate_phdr_cb __callback, void*__data);
+int dl_iterate_phdr(dl_iterate_phdr_cb_ngc __callback, void*__data) @nogc;
+int _rtld_addr_phdr(const void*, dl_phdr_info*) @nogc;
+int _rtld_get_stack_prot() @nogc;
```

Comment:
I wasn't referring to these, but it seems to be a good catch: [the `extern` here was meaningless as applied to functions](https://dlang.org/spec/declaration.html#extern)?

Reply:
Moved the extern(C) to the bottom to only pertain to these fout functions.

Reply:
No, move it back up, I wasn't talking about the attributes up top.  You need those there.